src/main/resources/migrations/00045_alter_table_gocardless_events.sql

```diff
@@ -0,0 +1,56 @@
+
```

Comment:
The file name should probably begin `00047` as we use a sequential prefix, looks like the next one is 47 maybe someone snuck in new files ahead of you :)